boards/shields/link_board_eth/link_board_eth.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_SPI=y
+CONFIG_SPI_1=y
```

Comment:
I know I let few of those go in previously (my bad, I'll get those cleaned), but: 
Theoretically, we should not enable board specific device instances.
Boards featuring a shield compatible connector (eg: arduino) should configure matching devices (eg: SPI_1 for spi_arduino) to be enabled if component is enabled, for instance:
```
if SPI
config SPI_1
    default y
endif
```
Because obviously, `CONFIG_SPI_1=y` won't fit for all boards featuring arduino connectors.

Reply:
Agree, but I am not sure if it is a solution for reel board. Moved it to `boards/shields/link_board_eth/boards/reel_board.conf`

Reply:
> I am not sure if it is a solution for reel board

Well indeed reel_board doesn't feature an arduino connector. 
But it could be other type of connector..